eng/targets/ResolveReferences.targets

```diff
@@ -252,20 +252,35 @@
       this assembly. Reset properties to avoid error when copying non-existent @(IntermediateRefAssembly) to
       $(TargetRefPath).
     -->
-    <GetTargetPathWithTargetPlatformMonikerDependsOn>$(GetTargetPathWithTargetPlatformMonikerDependsOn);AddReferenceProjectMetadata</GetTargetPathWithTargetPlatformMonikerDependsOn>
+    <GetTargetPathWithTargetPlatformMonikerDependsOn>
+      $(GetTargetPathWithTargetPlatformMonikerDependsOn);AddReferenceProjectMetadata
+    </GetTargetPathWithTargetPlatformMonikerDependsOn>
     <PrepareForRunDependsOn>RemoveReferenceProjectMetadata;$(PrepareForRunDependsOn)</PrepareForRunDependsOn>
   </PropertyGroup>
   <ItemGroup Condition=" $(HasReferenceAssembly) AND $(_CompileTfmUsingReferenceAssemblies) ">
-    <!-- Ensure ref/ project is built prior to the implementation project. Capture its target path. -->
+    <!-- Ensure ref/ project is built prior to the implementation project. -->
     <ProjectReference Include="$(_ReferenceProjectFile)">
-      <OutputItemType>ReferenceProjectMetadata</OutputItemType>
```

Comment:
/fyi depending on `%(OutputItemType)` metadata works most of the time but requires the ref/ project to build in some contexts where it isn't. This is more important when doing ApiCompat checks but might help in general.